scripts/release/create-canary-commands/npm-pack-and-unpack.js

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const {join} = require('path');
+const {exec} = require('child-process-promise');
+const {readdirSync} = require('fs');
+const {readJsonSync} = require('fs-extra');
+const {logPromise} = require('../utils');
+
+const run = async ({cwd, dry, tempDirectory}) => {
+  // Cleanup from previous build.
+  await exec(`rm -rf ${cwd}/build`);
+
+  // NPM pack all built packages.
+  // We do this to ensure that the package.json files array is correct.
+  const builtPackages = readdirSync(join(tempDirectory, 'build/node_modules/'));
+  for (let i = 0; i < builtPackages.length; i++) {
+    await exec(`npm pack ./${builtPackages[i]}`, {
+      cwd: `${tempDirectory}/build/node_modules/`,
+    });
+  }
+
+  await exec('mkdir build');
+  await exec('mkdir build/node_modules');
+  await exec(
+    `cp -r ${tempDirectory}/build/node_modules/*.tgz ${cwd}/build/node_modules/`
```

Comment:
this assumes /build/node_modules exists, ie - you've done a build previously on this machine. I just failed that (working on a fresh checkout). `mkdir -p`?

Reply:
(there's another instance of this, cmd-f cp -r)

Reply:
I'm not sure what this comment means. Literally the line right above this, I `mkdir build && mkdir build/node_modules`

Reply:
Oh! Ha! I forgot to specify `cwd` with this command.